src/include/mpir_ext.h.in

```diff
@@ -85,4 +85,17 @@ int MPIR_Ext_datatype_iscommitted(MPI_Datatype datatype);
 /* make comm split based on access to a common file system easier */
 int MPIR_Get_node_id(MPI_Comm comm, int rank, int *id);
 
+#ifdef ENABLE_QMPI
+extern void (**MPIR_QMPI_pointers) (void);
+extern void **MPIR_QMPI_storage;
+extern void (**MPIR_QMPI_tool_init_callbacks) (int);
+extern int MPIR_QMPI_num_tools;
+extern char **MPIR_QMPI_tool_names;
+extern int MPIR_QMPI_is_initialized;
+extern void (**MPIR_QMPI_first_fn_ptrs) (void);
+extern int *MPIR_QMPI_first_tool_ids;
+int MPII_qmpi_stash_first_tools(void);
+int MPII_qmpi_setup(void);
```

Comment:
why `mpir_ext.h`? Will ROMIO need it?

Reply:
Once the ROMIO bindings are getting automatically generated, yes.